test/actions/surveys.js

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
Do you mean the test file or the `eslint-disable`? Either way the answer is no. The these tests will more or less be the same once as add the remaining pieces. Maybe we will need to edit them as we add the pieces. And I set `eslint-disable` here to disable `no-unsed-expression` because it will throw a lint error if we try to use `expect.to.be.empty`.